frontend/packages/topology/src/filters/FilterProvider.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { useUserSettingsCompatibility } from '@console/shared';
+import { DEFAULT_TOPOLOGY_FILTERS } from './const';
+import { getAppliedFilters } from '../redux/action';
```

Comment:
`getAppliedFilters` was only used in this Provider. Can we remove it from `redux/action.ts` since it looks like that we still use redux here?!